sailfishusermanager.cpp

```diff
@@ -348,3 +349,65 @@ bool SailfishUserManager::hasAccessRights(uint uid_to_modify) {
 
     return true;
 }
+
+void SailfishUserManager::userSwitch(uint uid)
+{
+    if (!hasAccessRights(SAILFISH_UNDEFINED_UID)) {
+        auto message = QStringLiteral("Access denied");
+        qCWarning(lcSUM) << message;
+        sendErrorReply(QDBusError::AccessDenied, message);
```

Comment:
hasAccessRights already sends error replies.

Reply:
Perhaps it should be called `checkAccess` or something like that. `hasAccessRights` indeed sounds like a check of access rights without any side effects.

Reply:
Sounds like a good idea.